tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -1204,8 +1204,9 @@ class TestDataProcHiveOperator(unittest.TestCase):
     query = "define sin HiveUDF('sin');"
     variables = {"key": "value"}
     job_id = "uuid_id"
+    job_name = "simple"
     job = {
-        "reference": {"project_id": GCP_PROJECT, "job_id": "{{task.task_id}}_{{ds_nodash}}_" + job_id},
+        "reference": {"project_id": GCP_PROJECT, "job_id": f"{job_name}_{job_id}"},
```

Comment:
Adding the variable above `job_name` was the easiest solution I've found to avoid a considerable refactoring of the following tests. Any other suggestions are welcome

Reply:
Can you un-draft it please :)

Reply:
Done ! :). Thank you